credmark/types/data/block_number.py

```diff
@@ -140,16 +140,21 @@ def from_datetime(cls, datetime_date_ts: Union[datetime, date, int, float]):
         elif isinstance(datetime_date_ts, float):
             dt = datetime.fromtimestamp(int(datetime_date_ts), tz=timezone.utc)
         elif isinstance(datetime_date_ts, datetime):
-            dt = datetime_date_ts.replace(tzinfo=timezone.utc)
+            if not datetime_date_ts.tzinfo:
+                raise ModelInputError(
+                    f'Input datetime {datetime_date_ts} has no tzinfo, please add.')
+            ts = int(datetime_date_ts.timestamp())
         elif isinstance(datetime_date_ts, date):
```

Comment:
I fixed this. It should be assigning datetime_date_ts to dt and then is gets the ts from it below.